kotlin-wrappers-bom/build.gradle.kts

```diff
@@ -33,6 +33,7 @@ dependencies {
         api(project(":kotlin-ring-ui"))
         api(project(":kotlin-styled"))
         api(project(":kotlin-styled-next"))
+        api(project(":kotlin-tanstack-table-core"))
```

Comment:
Is it in alphabet order?

Reply:
Will fix it in next PR, because now it will produce merge conflict